hommod_rest/tasks.py

```diff
@@ -2,13 +2,28 @@
 
 from celery import current_app as celery_app
 
+from hommod_rest.services.modelutils import parseFasta, writeFasta
 from hommod_rest.services.model import modeler
-from hommod_rest.services.utils import list_models_of, select_best_model
+from hommod_rest.services.utils import (list_models_of, select_best_model,
+                                        get_oldest_hg_sequence, touch_file)
 
 _log = logging.getLogger(__name__)
 
 
 @celery_app.task()
+def remodel_oldest_hg():
+    fasta_path = get_oldest_hg_sequence()
+    with open(fasta_path, 'r') as f:
+        fasta = parseFasta(f)
+
+    sequence = fasta.values()[0]
+
+    touch_file(fasta_path)
```

Comment:
Why are you calling `touch_file`, which re-writes the file with its existing content?